vuln/npm/399.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": "399",
+  "title": "Command Injection - Generic",
+  "overview": "`whereis` concatenates unsanitized input into exec() command",
+  "created_at": "2018-02-25",
+  "updated_at": "2018-03-28",
+  "publish_date": "2018-03-28",
+  "author": "chalker (https://github.com/ChALkeR)",
+  "module_name": "whereis",
+  "cves": [
+    ""
+  ],
+  "vulnerable_versions": "<=0.4.0",
+  "patched_versions": ">=0.4.1",
+  "slug": "whereis-command-injection---generic",
+  "recommendation": "update whereis to 0.4.1 or higher",
```

Comment:
Whereis is deprecated by now, also 0.4.1 code is also not perfect (though significantly better).
0.5.0 is ok, but the package itself is deprecated.
I would change the recommendation to «use npm package `which` instead».